boards/google/quincy/google_quincy.dts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nuvoton/npcx9mfp.dtsi>
+#include <nuvoton/npcx/npcx9/npcx9-pinctrl.dtsi>
+
+/ {
+	model = "Google Quincy development board";
+	compatible = "google,quincy-fpmcu";
+
+	chosen {
+		zephyr,console = &uart1;
+		zephyr,shell-uart = &uart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,flash-controller = &int_flash;
+	};
+
+	aliases {
+		/* For watchdog sample */
+		watchdog0 = &twd0;
+	};
+};
+
+/* UART1: Servo UART (console) */
+&uart1 {
+	/* Use UART1_SL2 ie. PIN64.65 */
+	pinctrl-0 = <&uart1_2_sin_gp64
+				&uart1_2_sout_gp65>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+/* UART2: AP UART (Host Commands and MKBP) */
+&uart2 {
+	pinctrl-0 = <&uart2_sin_gp75
+			 &uart2_sout_gp86>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+/* SHI0: communication with the AP */
+ &shi0 {
+	status = "okay";
+	pinctrl-0 = <&shi_gp46_47_53_55>;
+	pinctrl-names = "default";
```

Comment:
hey there's few compliance warnings with this patch, you can run them locally with `./scripts/ci/check_compliance.py` if you don't want to wait for the CI run

Reply:
I'll reach out to you directly on this but when I try to run:
./scripts/ci/check_compliance.py -c upstream/main..
in a zephr venv, I get:
fatal: ambiguous argument 'upstream/main..': unknown revision or path not in the working tree.

In the meantime, I fixed the issue with the leading spaces in the dts file

Reply:
Your zephyr remote branch is probably called something different than "upstream" (likely "origin", I would guess)

Reply:
Thanks, Benjamin and Fabio, both:
./scripts/ci/check_compliance.py -c HEAD~1..HEAD
and
./scripts/ci/check_compliance.py -c origin/main..

work for me and all compliance warnings have been resolved in the local run